JASP-Engine/JASP/R/commonTTest.R

```diff
@@ -138,9 +138,9 @@
 
 .ttestVovkSellke <- function(table, options) {
   if (options$VovkSellkeMPR) {
-    message <-"Vovk-Sellke Maximum <em>p</em>-Ratio: Based on a two-sided <em>p</em>-value, 
-    the maximum possible odds in favor of H\u2081 over H\u2080 equals 1/(-e <em>p</em> log(<em>p</em>)) 
-    for <em>p</em> \u2264 .37 (Sellke, Bayarri, & Berger, 2001)."
+    message <-gettextf("Vovk-Sellke Maximum <em>p</em>-Ratio: Based on a two-sided <em>p</em>-value,
```

Comment:
This adds tabs at the start of the sentence. I don't think you want to do that. You can also see it in the .po file:

`"Vovk-Sellke Maximum <em>p</em>-Ratio: Based on a two-sided <em>p</em>-value,\n    the maximum possible odds in favor of H%1$s over H%2$s equals 1/(-e <em>p</em> log(<em>p</em>))\n    for <em>p</em> %3$s .37 (Sellke, Bayarri, & Berger, 2001)."`

Reply:
Apparently GitHub strips tabs, but you can look at your code changes

Reply:
Yes I  checked the result and that looked nice. But I changed it of course anyway.
Sorry I did it with an amend again, but you only need to check this Vovk text.